src/test/java/codeu/controller/AdminServletTest.java

```diff
@@ -59,28 +59,63 @@ public void setup() {
   public void testDoGet() throws IOException, ServletException {
 
     List<User> fakeUserList = new ArrayList<>();
-    User fakeUser = new User(UUID.randomUUID(), "test_username", "fakePasswordHash", Instant.now());
-    fakeUser.setAdmin(true);
+    User fakeUser = new User( UUID.randomUUID(), "test_username", "fakePasswordHash", Instant.now());
     fakeUserList.add(fakeUser);
 
     Mockito.when(mockUserStore.getAllUsers()).thenReturn(fakeUserList);
 
-    UUID fakeConversationId = UUID.randomUUID();
     List<Conversation> fakeConversationList = new ArrayList<>();
     fakeConversationList.add(
         new Conversation(UUID.randomUUID(), UUID.randomUUID(), "test_conversation", Instant.now()));
     Mockito.when(mockConversationStore.getAllConversations()).thenReturn(fakeConversationList);
 
     List<Message> fakeMessageList = new ArrayList<>();
     fakeMessageList.add(
-        new Message(UUID.randomUUID(),fakeConversationId, UUID.randomUUID(),"test_message",Instant.now()));
+        new Message(UUID.randomUUID(),UUID.randomUUID(), UUID.randomUUID(),"test_message",Instant.now()));
     Mockito.when(mockMessageStore.getAllMessages()).thenReturn(fakeMessageList);
+
     AdminServlet.doGet(mockRequest, mockResponse);
 
-    //Mockito.verify(mockRequestDispatcher).forward(mockRequest, mockResponse);
+    List<String> fakeAdminList = new ArrayList<>();
 
+    Mockito.when(mockSession.getAttribute("user")).thenReturn("test_username");
+    Mockito.when(mockUserStore.getUser("test_username")).thenReturn(fakeUser);
+
+      if(fakeUser.getIsAdmin() == true ){
```

Comment:
indentation

Reply:
also fakeUser.getIsAdmin() is always going to return false right. What is the point of this if block?